README.md

```diff
@@ -13,24 +13,55 @@ Please ensure your credentials are working by logging in. The platform link is s
+To install the `oc` tool, download the appropiate release from [Github](https://github.com/openshift/origin/releases). 
 
-Add those lines to your `~/.bashrc` or create a file with this content to be loaded by all users (in my case, i created `/etc/profile.d/user-bin-path.sh`):
-```
+#### Linux & Mac
+When you use Linux or Mac, exctract the `oc` and `kubectl` to a directory which is covered by your `PATH` variable. To ensure this happens, I use the following trick (tested on Ubuntu 16.04 and 18.04):
+
+Add those lines to your `~/.bashrc` or create a file with this content to be loaded by all users (Bast Practice: I created `/etc/profile.d/user-bin-path.sh`):
+```bash
 # Expand $PATH to include the .bin directory in the users home
 if [ -d "${HOME}/.bin" ]; then
     export PATH=$PATH:${HOME}/.bin/
 fi
 ```
 The create a folder called `.bin` in your home directory, e.g. via `mkdir ~/.bin`. Extract both filed mentioned above to this bin folder. Log out and log in again and check wether the new path got added to your `PATH` variable. You can check this by running `echo $PATH` from a command line. In that case you should no be able to issue a `oc` command.
 
+#### Windows
+When you are using Windows, make sure to have [Git for Windows](https://git-scm.com/download/win) installed. As part of the installation the Git Bash will be installed, which we will use for any commandline based action.
+
+First of all we need to create a .bash_profile, to save all of our additions to the `PATH` variable. Therefore simply enter `touch ~/.bash_profile`. Open it within you favorite text editor. 
+Here you will insert following content
+
+If you have to connect from behind a proxy remove the # in front of the proxy lines and add the appropriate proxyaddress.
+
+```bash
+#export HTTP_PROXY=http://<proxy_ip>:<proxy_port>
+#export HTTPS_PROXY=https://<proxy_ip>:<proxy_port>
+
+export PATH=/C/Users/<YourUser>/<Projectpath>/oc-client:$PATH
+```
+Now exctract the `oc` and `kubectl` to the directory which we have just covered by the `PATH` variable.
+
 ### Adding autocompletion
 
-The `oc` tools already supplies you with a bash script that can be used for autocompletion. For my Ubuntu System I used `oc completion bash > ~/oc-completion` and put the result file to `etc/bash_completion.d`. After logging in again, I was now able to use oc autocompletion
+The `oc` tools already supplies you with a bash script that can be used for autocompletion. Use following line to generate a bashscript which will offer auto completion.
+
+```bash
+oc completion bash > ~/<project-path>/oc-completion.sh
+```
+
+Now we will add another line to our `.bashrc`/`.bash_profile` (accodring to your setup).
+
+Line to add:
+```bash
+source ~/<project-path>/oc_completion.sh
```

Comment:
We should rather check if the file exists and only in this case source it. otherweise the user will get an error each time he logs in. See [here](https://www.electrictoolbox.com/test-file-exists-bash-shell/) for examples

Reply:
Added Wrapper around it.